test/integration/firestore.js

```diff
@@ -47,29 +47,34 @@ function test(utils) {
             utils.logSuccess('firestore()');
         })
         .catch(err => {
-            utils.logFailure('firestore()', 'Error while interacting with Firestore: ' + err);     
+            utils.logFailure('firestore()', 'Error while interacting with Firestore: ' + err);
         });
   }
 
   function testFieldValue() {
-    const expected = {
-        name: 'Mountain View',
-        population: 77846,
-        timestamp: admin.firestore.FieldValue.serverTimestamp(),
-    };
     const reference = admin.firestore().collection('cities').doc()
-    return reference.set(expected)
+    return Promise.resolve()
```

Comment:
What was the rationale behind this change?

It seems like you've added an extra step in the chain unnecessarily.

Reply:
This is to allow any errors thrown by `admin.firestore.FieldValue` to be caught in the `catch` block. I noticed that errors thrown outside the promise chain, cause the test suite to come to a halt.

Reply:
Ohhh gotcha. Ok, then yeah this is fine then.